### PR TITLE
CNF-14958: Update readme & examples for changing the CI defaults ConfigMap

### PR DESCRIPTION
**Pull request by irinamihai**
Description:
- update an annotation instead of adding a route

### CODE REVIEW

File: docs/cluster-configuration.md [hunk 1]
```diff
@@ -195,14 +195,12 @@ The following steps need to be taken:
...
-    * **Note:** Depending on the changes in the default `ConfigMap`, updates to the `spec.templateParameters.clusterInstanceParameters` of the `ProvisioningRequest` might be needed.
 4. The O-Cloud Manager detects the change:
-    * It updates the `ClusterInstance` with the new route.
+    * It updates the `ClusterInstance` with the new annotation.
 5. The siteconfig operator detects the change to the `ClusterInstance` CR:
-    * It updates the `NMStateConfig` installation manifest to contain the new route.
-    * **Note:** Some installation manifests cannot be updated after provisioning as the underlying operators have webhooks to prevent such updates.
-6. The change is rolled out to the `ManagedCluster`.
+    * The new annotation is added to the `ManagedCluster`.
     * Any issues are reported in the `PrivisioningRequest`, under `status.conditions`.
```
Comment (browsell) [inline docs/cluster-configuration.md:202]: nit, previous typo PrivisioningRequest

**Review by Missxiaoguo — COMMENTED** [review]
/lgtm

**Pull request merged**